docs/testing/continuous-integration.md

```diff
@@ -1,43 +1,62 @@
+* Civi Tests - These are tests that mainly test the `Civi()::` facade. They can be found in the `tests/phpunit/civi` directoy of civicrm-core repo.
+* E2E Tests - These are tests that test the integration with the Content Management System (CMS) they usually involve testing rest or Soap interfaces. These can be found in `tests/phpunit/E2E` directoy of the civicrm-core repo
+* Upgrade Tests - This is a test where it takes a known databasse state loads it into the test db and then runs the upgrade script to try and upgrade the database to the current code version.
+* Webtests  - These are selenium webtests that are run occassionally and haven't been well maintained
 
-For new (unrecognised by Jenkins) contributors, Jenkins will automatically
-respond "can an admin verify this patch?", and a Github user with admin
-permissions may approve running the tests by commenting on the PR "ok to test".
+To run all the tests in one of the suites locally, you can use [civi-test-run](/tools/civi-test-run.md).
 
-If the user is trusted, CiviCRM administrators can add the person to the
-whitelist by commenting "add to whitelist".
+The tests that are run are variable depending on the repository that is triggered.
 
-## Build Schedule
+* `civicrm-core`: Upgrade, karma, CRM, api_v3, E2E and Civi tests
+* `civicrm-packages`: same as civicrm-core
+* `civicrm-drupal`: Upgrade, Karma and E2E tests
+* `civicrm-backdrop`: same as civicrm-drupal
+
+For PR test jobs against `civicrm-core` and `civicrm-packages`, Jenkins only builds a Drupal site to run the tests against. If you are fixing an issue with another CMS you may need to build yourself a local test environment with that CMS
+
+If the tests have failed for something that we suspect is a random failure, we can ask Jenkins to run the tests again by commenting in the PR "Jenkins, test this please"
+
+## Matrix Test Jobs
+
+The other type of job that Jenkins runs is what is described as a "matrix" job. This is a much more extended version of the PR job and is usually run against multiple webserver configurations.
+
+The main difference between the `CiviCRM-Core-Matrix` job and a PR test job is that it runs more upgrade tests than the `CiviCRM-Core-PR`.
+
+The other matrix job is one that runs the webtests.
+
+The matrix jobs operate on two main combinations, a PHP5.5 + MySQL5.5 server and a PHP7.0 + MySQL5.7 test seerver.
+
+Due to the length of the full test suites, matrix jobs can take from 2 to 24 hours to complete.
+
+### Build Schedule
+
+Jenkins periodically runs the full suite (once every 4-24hr) on the official codebase.
 
-Jenkins periodically runs the full suite (once every 4-24hr) on the official 
-codebase.
- 
 Code-style is not checked on this build, but all upgrade and web tests are run.
 
-## Running the test suite locally
 
-You can use [civi-test-run](/tools/civi-test-run.md) locally to run a full standard CiviCRM Test suite.
+## Jenkins Whitelist
```

Comment:
I'd lean toward putting the whitelist info higher in the doc -- from the perspective of a new reader/contributor, you'll need to get whitelisted before you start analyzing/understanding the test-results on your Pr.